actsims/signal.py

```diff
@@ -76,7 +73,7 @@ def get_base_alm_idx(self, set_idx, sim_num):
     def get_signal_idx(self, season, patch, array, freq, set_idx, sim_num):
         return '_'.join([season, patch, array, freq, 'set0%d'%set_idx, '%05d'%sim_num])
 
-    def get_signal_sim(self, season, patch, array, freq, set_idx, sim_num, save_alm=False, save_map=False,oshape=None,owcs=None):
+    def get_signal_sim(self, season, patch, array, freq, set_idx, sim_num, oshape, owcs, save_alm=False, save_map=False):
```

Comment:
Do we want to make oshape and owcs optional such that it defaults to the full footprint from the mapmaker? @ajvanengelen @dwhan89 up to you based on use cases

Reply:
Wouldn't that lead to the same hard-to-understand assertion failure that started all of this, if someone tried to do both signal and noise?

Reply:
Yeah, I think I like it as it is. Let's just force users to provide wcs/shape

Reply:
Sounds good.